.openshift-tests-extension/openshift_payload_cluster-version-operator.json

```diff
@@ -1,12 +1,27 @@
 [
   {
-    "name": "[Jira:Cluster Version Operator] cluster-version-operator-tests should support passing tests",
+    "name": "[Jira:\"Cluster Version Operator\"] cluster-version-operator-tests should support passing tests should support passing tests",
```

Comment:
Please note when deleting and recreating the metadata file in a separate commit. The file should only be updated via the `make update` normally. We are intentionally bypassing this as of the moment, but that should be clearly communicated in a commit description as in "we know what we are doing, in the future, do not do that, do it properly or else history is lost"

Reply:
I will be updating the README file to clearly state this for the future.